Adyen/UI/Form/Items/Error/FormErrorItemStyle.swift

```diff
@@ -0,0 +1,32 @@
+//
+// Copyright (c) 2021 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import UIKit
+
+/// Contains the styling customization options for an error item in a form.
+public struct FormErrorItemStyle: ViewStyle {
+
+    /// The message style.
+    public var message = TextStyle(font: .preferredFont(forTextStyle: .body),
+                                   color: UIColor.Adyen.componentLabel,
+                                   textAlignment: .natural)
+
+    /// The corners style of the text item.
+    public var cornerRounding: CornerRounding = .fixed(6)
+
+    /// :nodoc:
+    public var backgroundColor = UIColor.Adyen.errorRed.withAlphaComponent(0.1)
```

Comment:
Just wondering:

can we include `withAlphaComponent(0.1)` into `errorRed` and call it `errorBackgroundRed` ?

Reply:
I prefer not, because in other places different alpha might be applied